README.md

```diff
@@ -27,15 +27,13 @@ Start the server
 
 Start the bot
 
-    $ curl -d "{\"discord_token\":\"yourtoken\"" \ 
-           -h "content-type: application/json" \
-           -x post http://localhost:80/api/v1/start 
-           
+    curl -d "discord_token=$DISCORD_TOKEN" -X POST \
+         http://localhost:3000/api/v1/start
+
 Stop the bot
 
-    $ curl -d "{\"discord_token\":\"yourtoken\"" \ 
-           -h "content-type: application/json" \
-           -x post http://localhost:80/api/v1/stop 
+    curl -d "discord_token=$DISCORD_TOKEN" -X POST \
+         http://localhost:3000/api/v1/stop
            
 ## Development
```

Comment:
Add some steps instructing the user on how to use the formatting stuff you added.